docs/logs/discovery/control-plane.md

```diff
@@ -0,0 +1,32 @@
+title: Logs Discovery Control Plane
+description: Logs Discovery Control Plane
+
+Logs Discovery Control Plane is the central location for managing the shipping of discovered log sources. It consists of various tabs, each of them designated to one particular log source.
+
+The rows are comprised of the following columns:
+
+- `Name` represents the human-friendly identifier for the log source, such as log file name or container name.
+- `Infra App` is linked to the App token that was used to deploy the Sematext Agent. The Infra App stores infrastructure data and metrics including discovered log sources.
```

Comment:
I think this one will sometimes appear and sometimes not, maybe mention that? Also not sure how clear this explanation of `Infra App` will be to users.

Reply:
> maybe mention that?

right.

Got a better explanation of the Infra App?